### ISSUE TITLE
Accessibility issue - lack of alt text and unlabelled buttons

### ISSUE THREAD

**Issue by pauljonesOH** [mattermost/mattermost-server#8446]
I'm logging some accessibility issues in Mattermost that are effecting the usability for a blind user. 

#### Summary
Lack of alt text and unlabelled buttons

#### Steps to reproduce
(Mattermost 4.4.2 using JAWS screen reader software)

#### Expected behavior
Everything should be labelled so that it gives alt text to the screen reader

#### Observed behavior (that appears unintentional) 
For example, the file upload/attach file button doesn't have alt text

#### Possible fixes
Apply alt text wherever possible

**Comment by lindy65** [comment]
Thanks for the report @pauljonesOH :)

I've created [this ticket](https://mattermost.atlassian.net/browse/MM-9820) for us to fix the issue.

**Comment by lindy65** [comment]
I'll close this issue off here for now and we can track via the Jira ticket linked above.
